tests/integration/src/test/java/org/mule/test/construct/FlowConfigurationFunctionalTestCase.java

```diff
@@ -77,27 +79,27 @@ public void testFlowSynchronous() throws Exception {
     assertNotNull(message);
     Thread thread = (Thread) message.getPayload().getValue();
     assertNotNull(thread);
-    assertEquals(Thread.currentThread(), thread);
+    assertEquals(currentThread(), thread);
   }
 
   @Test
   public void testFlowAynchronous() throws Exception {
-    flowRunner("asynchronousFlow").withPayload("0").asynchronously().run();
+    flowRunner("asynchronousFlow").withPayload("0").withExchangePattern(ONE_WAY).dispatch();
```

Comment:
Switched to used `run()`, as test doesn't depend on dispatch.  I'll review other uses of dispatch to see if any can be changed.